cdap-ui/app/cdap/components/Pagination/Pagination.less

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright © 2016 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+.pagination-container {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  overflow-x: hidden;
+
+  .pagination-content {
+    display: inline-block;
+    width: 100%;
+  }
+
+  .change-page-panel {
+    display: inline-block;
+    position: absolute;
+    cursor: pointer;
+
+    /*Page height minus footer height*/
+    height: ~"-moz-calc(100vh - 50px)";
+    height: ~"-webkit-calc(100vh - 50px)";
+    height: ~"calc(100vh - 50px)";
+    z-index: 300;
+    padding-left: 15px;
+    padding-right: 15px;
+    opacity: 0;
+    padding-left: 5px;
+    padding-right: 5px;
+
+    &:hover,
+    &.pressed {
+      opacity: 0.8;
+    }
+  }
+  .change-page-panel-left {
+    position: absolute;
+    left: 0;
+  }
+  .change-page-panel-right {
+    position: absolute;
+    right: 0;
+  }
+  .page-change-arrow-container {
+    position: relative;
+    top: 50%;
+    transform: scale(1.5,1);
```

Comment:
Do we need this? It actually stretches the icon vertically which looks a little out of place. 

<img width="46" alt="screen shot 2016-11-02 at 11 24 21 pm" src="https://cloud.githubusercontent.com/assets/1452845/19957380/84b2bb80-a153-11e6-8053-b3fabe3a6c77.png">

vs 

<img width="63" alt="screen shot 2016-11-02 at 11 24 15 pm" src="https://cloud.githubusercontent.com/assets/1452845/19957381/871aaff4-a153-11e6-9088-2058249ac8f1.png">